core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -288,6 +298,13 @@ class KafkaController(val config: KafkaConfig,
 
   private def state: ControllerState = eventManager.state
 
+  private def timing(taskStartMs: Long, methodStartMs: Long): String = {
+    val nowMs = time.milliseconds()
+    if (taskStartMs < 0) s"(elapsed init = ${nowMs - methodStartMs} ms)"
```

Comment:
At least as I've used it, one adds the -1 manually at the call site, so it would be totally meaningless to intentionally call it with two -1's (or negative values in general).  And it's not like it will throw an exception if you do; you'll just get a very large and presumably very obviously bogus result.